modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,106 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-704]
+== Release 7.0.4 (May 2022)
+Couchbase Server 7.0.4 was released in May 2022.
+This maintenance release contains fixes to issues.
+
+[#fixed-issues-704]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
+
+=== Query Service
+
+[#table_fixedissues_v704-queryservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-48438[MB-48438^]
+| *Summary*: Include bucketType in pools/default/b/<bucket-name> REST API
+
+| https://issues.couchbase.com/browse/MB-51581[MB-51581^]
+| *Summary*: JOIN query not covered
+
+| https://issues.couchbase.com/browse/MB-51453[MB-51453^]
+| *Summary*: IN/NOT IN filters not using Hash for evaluation
+
+| https://issues.couchbase.com/browse/MB-51431[MB-51431^]
+| *Summary*: Avoid circular link in annotated value hierarchy
+
+| https://issues.couchbase.com/browse/MB-51447[MB-51447^]
+| *Summary*: select executing a function executing a select returns the same results
+
+| https://issues.couchbase.com/browse/MB-51739[MB-51739^]
+| *Summary*: LEFT JOIN breaks with between operator on non-existing attribute
+
+| https://issues.couchbase.com/browse/MB-51578[MB-51578^]
+| *Summary*: Error 4321 generated when OR clause has a ($parameter is null) arm
+
+|===
+
+=== Analytics Service
+
+[#table_fixedissues_v704-analyticsservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
+
+| https://issues.couchbase.com/browse/MB-51483[MB-51483^]
+| *Summary*: Data is not completely ingested into datasets
+
+| https://issues.couchbase.com/browse/MB-51482[MB-51482^]
+| *Summary*: Node is not upgraded and stuck trying to register with CC
```

Comment:
Do we add release notes for backports? (MB-51482 and MB-51488)?

Reply:
I'm open to your guidance on this.

Reply:
If  issues are included in the filter output, then they are considered candidates. That is the process.